atecc608a_se.c

```diff
@@ -140,7 +139,7 @@ static void pubkey_for_psa(uint8_t *data)
 static psa_status_t is_public_key_slot(uint16_t key_slot)
```

Comment:
Preexisting: `is_xxx` strongly evokes a function that returns a boolean. A name with `check` or `validate` would better convey that the function returns a status. Alternatively, make the function return a boolean.